bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -471,7 +471,11 @@ spec:
         {{- end }}
         resources:
           requests:
+            {{- if .Values.persistence.volumeClaimTemplates.requests }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.persistence.volumeClaimTemplates.requests "context" $) | nindent 12 }}
+            {{- else }}
```

Comment:
such as storage already appears I think it's confusing to have in this way, for me when I read custom PVC requests attributes are attributes no appears by default, and storage already appears.
```diff
   resources:
            requests:
              storage: "20Gi"
+             iops: "1000"
```
Could you implement it without storage are in `persistence.volumeClaimTemplates.requests`?

Reply:
@Mauraza Good point. I've made the change you suggested